django/thunderstore/api/urls.py

```diff
@@ -0,0 +1,36 @@
+from django.urls import path
+
+from thunderstore.api.cyberstorm.views import CommunitiesAPIView, CommunityAPIView, PackageAPIView, PackagePreviewAPIView, TeamAPIView, UserAPIView
+
+urls = [
+    path(
+        "c/",
+        CommunitiesAPIView.as_view(),  # type: ignore
```

Comment:
If the type-ignoring is really needed, could it be done once on the import row?

Reply:
I've been adding the type ignores everywhere, where it took me more than 5 seconds to come up with the type. Not super used to typed python and didn't want to spend too much time on these type corrections.

I'll add it to my minor list of stuff to do; to go through these.

Reply:
You should not use type ignore haphazardly like that. We're not enforcing typechecks on the project yet, but if we were to do that, this would now get ignored which is not something that should happen.

If you want to ignore them, do it in a way that only applies to your own dev environment rather than globally to the project.